eks/core.py

```diff
@@ -259,6 +261,48 @@ def compute_nll(innovations, innovation_covs, epsilon=1e-6):
 
 @typechecked
 def jax_ensemble(
+    marker_array: MarkerArray,
+    avg_mode: str = 'median',
+    var_mode: str = 'confidence_weighted_var',
+) -> MarkerArray:
+    """
+    Compute ensemble mean/median and variance of a marker array using JAX.
+    """
+    n_models, n_cameras, n_frames, n_keypoints, _ = marker_array.array.shape
```

Comment:
minor: I see you have a "MarkerArray.shape()" method; if you turn this into a getter method then you can just  call "marker_array.shape" here